src/main/scala/ba/sake/hepek/bulma/component/DropdownComponents.scala

```diff
@@ -37,7 +37,7 @@ case class DropdownMenu(elements: DropdownElement*) extends BulmaElement {
 
 trait DropdownComponents {
 
-  def dropdown(trigger: Option[DropdownTrigger], menu: DropdownMenu) =
+  def dropdown(trigger: Option[DropdownTrigger] = None, menu: DropdownMenu) =
```

Comment:
Same here, non-default parameters should go first if there's no reason to not put them first.

Reply:
Same as before, the parameters have been ordered according to the Bulma documentation.